traitator/tests/test_qctraits.py

```diff
@@ -20,14 +20,14 @@ def _find_nan_rows(table: pd.DataFrame, cols: [str]) -> pd.DataFrame:
 class QCTests(TestCase):
     def setUp(self):
         self.fp_guitar = 'traitator/tests/data/trait_data_n10906.csv'
-        self.fp_tresor = 'data/traitdata_tresor.csv'
+        self.fp_tresor = 'data/Traits_v2.csv'
         self.cols_binomial = ['Genus', 'Species']
 
     def tearDown(self):
         pass
 
     def test_qctresor(self):
-        tresor = pd.read_csv(self.fp_tresor, sep="\t")
+        tresor = pd.read_csv(self.fp_tresor, sep=",")
```

Comment:
you should not touch the test code other than adding additional tests! If you are using a different separator, please change on your side - not in our shared code!